homeassistant/components/statistics/sensor.py

```diff
@@ -162,70 +177,79 @@ def valid_binary_characteristic_configuration(config):
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the Statistics sensor."""
 
     await async_setup_reload_service(hass, DOMAIN, PLATFORMS)
 
     async_add_entities(
-        [
+        new_entities=[
             StatisticsSensor(
-                source_entity_id=config.get(CONF_ENTITY_ID),
-                name=config.get(CONF_NAME),
-                unique_id=config.get(CONF_UNIQUE_ID),
-                state_characteristic=config.get(CONF_STATE_CHARACTERISTIC),
-                samples_max_buffer_size=config.get(CONF_SAMPLES_MAX_BUFFER_SIZE),
-                samples_max_age=config.get(CONF_MAX_AGE),
-                precision=config.get(CONF_PRECISION),
-                quantile_intervals=config.get(CONF_QUANTILE_INTERVALS),
-                quantile_method=config.get(CONF_QUANTILE_METHOD),
+                source_entity_id=config[CONF_ENTITY_ID],
+                name=config[CONF_NAME],
+                unique_id=config.get(CONF_UNIQUE_ID, None),
```

Comment:
`None` is the default parameter to `dict.get`, I think we can just drop it?
Same for `samples_max_age`.